<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>801</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QListWidget" name="specialitiesList"/>
    </item>
    <item>
     <widget class="QListWidget" name="groupsList"/>
    </item>
    <item>
     <widget class="QListWidget" name="studentsList"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>801</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAdd_New">
    <property name="title">
     <string>Database</string>
    </property>
    <addaction name="actionAdd_New"/>
   </widget>
   <addaction name="menuAdd_New"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAdd_New">
   <property name="text">
    <string>Add New</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
